addons/udes_stock_picking_batch/models/stock_picking_type.py

```diff
@@ -1,5 +1,17 @@
-from odoo import models
+from odoo import models, fields
 
 
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
+
+    u_auto_assign_batch_pick = fields.Boolean(
+        string="Auto assign picks when added to a running batch",
```

Comment:
Let's cut this down, it won't display nicely on the form view.

Maybe something like: `Auto Assign Running Batch Picks`?

We have the help text for a more detailed explanation.

Reply:
Done